packages/web/src/components/edit/fields/DatePickerField.module.css

```diff
@@ -14,6 +14,12 @@
   margin-bottom: var(--harmony-unit-1);
 }
 
+.noValue {
+  font-size: var(--harmony-font-l);
+  line-height: 18px;
```

Comment:
nit: this shouldn't be necessary..

Reply:
no? for date picker it seemed there was no good default when there is no value, which we want with the new designs. ideally we move this all to text-input though!